src/Illuminate/Foundation/Testing/CrawlerTrait.php

```diff
@@ -76,6 +76,28 @@ public function get($uri, array $headers = [])
     }
 
     /**
+     * Visit the given URI with a GET request with content type of application/json.
+     *
+     * @param  string  $uri
+     * @param  array  $data
+     * @param  array  $headers
+     * @return $this
+     */
+    public function getJson($uri, array $data = [], array $headers = [])
+    {
+        $content = json_encode($data);
+        $headers['CONTENT_TYPE'] = 'application/json';
+        $headers['CONTENT_LENGTH'] = strlen($content);
+        $headers['Accept'] = $headers['Accept'] ?: 'application/json';
```

Comment:
It's not a fatal error (at least not in any version of PHP that Laravel supports), but it is potentially an "undefined index" notice.

I'll update that, but I feel that an expected 'Accept' header for json calls is appropriate (and easy to overwrite).